Minesweeper/Minesweeper/Entities/CellState.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CellState.swift
+//  Minesweeper
+//
+//  Created by Dorian Grolaux on 20/02/2018.
+//
+
+import Foundation
+
+enum CellState {
```

Comment:
Could this be a private member of Cell? I can't see it being used outside of the Cell.

Reply:
It's defo going to be used for the UI to decide which sprite to display for instance.